File: C1.UWP.FlexChart/CS/StockChart/StockChart/Converters/IsComparisonModeToDisplayModeConverter.cs
using System;
using System.Collections.Generic;
using System.Globalization;
using System.Linq;
using System.Text;
using Windows.UI.Xaml.Data;

namespace StockChart.Converters
{
    public class IsComparisonModeToDisplayModeConverter : IValueConverter
    {
        public object Convert(object value, Type targetType, object parameter, string language)
        {
            bool b = System.Convert.ToBoolean(value);
            return b ? DisplayMode.Comparison : DisplayMode.Independent;
        }

        public object ConvertBack(object value, Type targetType, object parameter, string language)
        {
            DisplayMode dm = (DisplayMode)value;
            return dm == DisplayMode.Comparison;
        }
    }
}

File: C1.UWP.FlexChart/VB/StockChart/StockChart/Converters/IsComparisonModeToDisplayModeConverter.vb
Public Class IsComparisonModeToDisplayModeConverter
    Implements IValueConverter
    Public Function Convert(value As Object, targetType As Type, parameter As Object, language As String) As Object Implements IValueConverter.Convert
        Dim b As Boolean = System.Convert.ToBoolean(value)
        Return If(b, DisplayMode.Comparison, DisplayMode.Independent)
    End Function

    Public Function ConvertBack(value As Object, targetType As Type, parameter As Object, language As String) As Object Implements IValueConverter.ConvertBack
        Dim dm As DisplayMode = DirectCast(value, DisplayMode)
        Return dm = DisplayMode.Comparison
    End Function
End Class